platform/nativescript/framework.js

```diff
@@ -19,27 +13,6 @@ setVue(Vue)
 Vue.use(ModalPlugin)
 Vue.use(NavigatorPlugin)
 
-const newLineRegExp = /\\n/g
-
-console.log = (function(log, inspect, Vue) {
-  return function(...args) {
```

Comment:
@farfromrefug I'm not too keen on removing this - I'd rather make it opt-out, as I've had a much better development experience with this in place vs. the default console.log.

Reply:
@rigor789 ok i understand. Make it an opt-in then. GDPR ;) It still crashes for me + i don't really want it. And users won't understand why. If you can't recall why it was necessary then i think it really should be opt-in and not opt-out

Reply:
@farfromrefug the reason I had to add it was that the built-in console.log crashed the app for anything more "complex" than a string, or a simple object `{ foo: "bar" }`. Most notably it crashed whenever trying to log a vue observed property, which was probably 90% of the things you'd log.

Plus it adds colors - which I really like when looking at the logs, makes it a lot easier to find things (this might be just my preference, and it's disabled by default on the playground anyways)

Reply:
@rigor789 those crashes do not appear anymore. Plus the contrary happens. Crashes with it.
If you use it just for the colors, should be in userland. plus it adds unecessary dependencies in production.

Reply:
@rigor789 so we can keep this in userland?